src/tm/class-wpml-tm-page-builders-field-wrapper.php

```diff
@@ -94,8 +94,21 @@ public function get_string_type() {
 	}
 
 	/**
-	 * @param int $package_id
-	 * @param int $string_id
+	 * Get string wrap tag.
+	 *
+	 * @param stdClass $string WPML string.
+	 *
+	 * @return string
+	 */
+	public static function get_wrap_tag( $string ) {
```

Comment:
I still don't understand the need for this method. Did I miss something @kagg-design?

Reply:
@strategio It is used in wp-content/plugins/sitepress-multilingual-cms/vendor/wpml/page-builders/src/tm/class-wpml-tm-page-builders.php:78